pkg/controller/releasemanager/releasemanager_controller.go

```diff
@@ -172,6 +172,28 @@ func (r *ReconcileReleaseManager) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
+	deliveryClusters, err := r.getClustersByFleet(ctx, rm.Namespace, r.config.ServiceLevelsFleet)
```

Comment:
Is this to support setting up new `delivery` clusters during a new edition migration?

Reply:
Yeah and to make it generally configurable.